woocommerce/woocommerce
# Update blocks build message

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/44406

This PR updates the instructions text when the Blocks assets are not built.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout a fresh copy of this repo (then checkout this branch) or simply delete the build files from `/plugins/woocommerce/assets/client/blocks`
2. Go to the backend and ensure you're seeing the updated correct build instructions on top.
![Screenshot by Dropbox Capture](https://github.com/woocommerce/woocommerce/assets/2132595/6a89f3f4-3ecb-425d-b811-25124679578b)



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

Update Blocks build message when asset files are not built.
</details>
